### ISSUE TITLE
Improve support for CI/CD

### ISSUE THREAD

**Issue by imbrn** [imbrn/v8n#190]
_Problem:_ Nowadays, we have to execute the library deployment process every time we need to produce a new version.

It would be nice to have a complete CI/CD process, where every new merge on the main branch would produce an automatic deployment.

Maybe we should adhere to the [semantic-release](https://github.com/semantic-release/semantic-release) project for helping us to achieve a commit-based versioning system. :thinking:

**Comment by barfurth** [comment]
Good idea, would target 1.5.0 with this. So once 1.4.0 is released we use the commit format?
